src/lib/metro/common.ts

```diff
@@ -1,13 +1,50 @@
-import { DiscordStyleSheet } from "@types";
-import { find, findByProps } from "@metro/filters";
+import { find, findByProps, findByPropsAll, findByStoreName } from "@metro/filters";
+import type { StyleSheet } from "react-native";
+import { ReactNative as RN } from "./common";
+
+const ThemeStore = findByStoreName("ThemeStore");
+const colorResolver = findByProps("colors", "meta").meta;
+
+// Reimplementation of Discord's createThemedStyleSheet, which was removed since 204201
+// Not exactly a 1:1 reimplementation, but sufficient to keep compatibility with existing plugins
+function createThemedStyleSheet<T extends StyleSheet.NamedStyles<T>>(sheet: Styles<T>) {
+    Object.values(sheet).forEach(s => {
+        const style = RN.StyleSheet.flatten(s);
+
+        Object.keys(style).forEach((key) => {
+            if (colorResolver.isSemanticColor(style[key])) {
+                const symbol = style[key];
```

Comment:
Does this prevent styles of unloaded plugins from being garbage collected, if so, is it possible to circumvent this using `WeakRef` or something?

Reply:
Yes, and no, Hermes shipped doesn't include WeakRef support

Reply:
Pushed an alternative approach that should resolve this